06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,142 @@
+const invertedIndex = new InvertedIndex();
+
+
+const Book = require('./book.json');
```

Comment:
You have eslint violations on this file. Please address them.
- Line length limit (80 characters max)
- Missing semicolons
- Trailing spaces

Please address these and others... thanks